src/backend/distributed/executor/multi_utility.c

```diff
@@ -391,6 +399,25 @@ multi_ProcessUtility(Node *parsetree,
 	{
 		ListCell *ddlJobCell = NULL;
 
+		if (IsA(parsetree, AlterTableStmt))
```

Comment:
Can we add a comment explaining why we have these checks here instead of earlier with other checks?